__tests__/list-test.js

```diff
@@ -0,0 +1,13 @@
+import 'react-native';
+// import React from 'react';
+// import EventList from '../shared/components/EventList';
+
+// Note: test renderer must be required after react-native.
+// import renderer from 'react-test-renderer';
+
+it('renders correctly', () => {
+  // const tree = renderer.create(
+  //   <EventList />
+  // ).toJSON();
+  // expect(tree).toMatchSnapshot();
```

Comment:
it's not work properly. 'fetch' is not work in this test_case. I should figure out the issue.